themes/theme-gmd/components/Search/index.jsx

```diff
@@ -6,7 +6,7 @@ import { SEARCH_PATTERN } from '@shopgate/pwa-common-commerce/search/constants';
 import AppBar from './components/AppBar';
 import Backdrop from './components/Backdrop';
 import Suggestions from './components/Suggestions';
-import { TOGGLE_SEARCH } from './constants';
+import { TOGGLE_SEARCH, SEARCH_CLOSED } from './constants';
```

Comment:
Could you check `toggle` subscription? We subscribe in constructor, but i dont see, that subscription is removed when component is unmounted.

Reply:
Done